tests/plugins/generic/test_ddpm.py

```diff
@@ -19,11 +19,16 @@
 plugin_name = "ddpm"
 plugin_params = dict(
     n_iter=1000,
-    batch_size=200,
-    num_timesteps=500,
-    log_interval=10,
-    print_interval=100,
+    batch_size=1000,
+    num_timesteps=100,
+    model_type="mlp",
 )
+# plugin_params = dict(
+#     n_iter=1000,
+#     batch_size=1000,
+#     num_timesteps=30,
+#     model_type="tabnet",
```

Comment:
Why is this commented?

Reply:
This was for testing the usage of TabNet as the diffusion model of TabDDPM. It fitted successfully but the performance was now as good as MLP.